spec/core_functions/color/hsl/multi_argument_var.hrx

```diff
@@ -36,6 +29,11 @@ a {
 
 <===>
 ================================================================================
+<===> 1_of_2/options.yml
+---
+:todo:
+- libsass
```

Comment:
Can you add issue references for any of these TODOs that don't have them yet?

Reply:
No I can't without spending substantial time to find them.
I should have preserved those that had references.
Or did I remove any of them?

Reply:
If you don't know of an existing issue, can you file a new one? I understand that this came from an existing TODO without an issue reference, but since you're touching it anyway we should update it to point to the right place.